ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -136,7 +152,32 @@ ompi_mtl_ofi_component_register(void)
                                     MCA_BASE_VAR_SCOPE_READONLY,
                                     &ompi_mtl_ofi.ofi_progress_event_count);
 
-     free(desc);
+    free(desc);
+
+    ret = mca_base_var_enum_create ("ofi_tag_mode_type", ofi_tag_mode_type , &new_enum);
+    if (OPAL_SUCCESS != ret) {
+        return ret;
+    }
+
+    ofi_tag_mode = MTL_OFI_TAG_AUTO;
+    asprintf(&desc, " Mode for OFI tag."
+            " 1 auto (default): detect if the provider supports FI_REMOTE_CQ_DATA or fallback to 2."
```

Comment:
I'm not quite sure what the explanation for 1 means.  Does it mean:

1. If FI_REMOTE_CQ_DATA is supported, use 4.
2. Otherwise, use 2.
3. 3 is available just for the heckuvit (i.e., different bit counts than 2)
4. How many bits are used for 4?  I only ask because they're specified / shown for 2 and 3, but not 4 (or 1).

Reply:
All answers are correct.  4 or 1 do not show this info because there are no limitations: 32b. Should it still show it? or at least mention there are no limitations using it.